src/app/components/experiment-timeline/experiment-timeline.component.html

```diff
@@ -53,6 +53,15 @@ <h1 class="t-page-headline align-arrow">Experiment Timeline</h1>
                 [value]="unclearedSubstepValue.value">{{unclearedSubstepValue.viewValue}}
             </mat-radio-button>
         </mat-radio-group>
+
+        <div class="radio-label">
+            <mat-label>Result quality</mat-label><br>
+        </div>
+        <mat-radio-group [(ngModel)]="resultQuality" (ngModelChange)="updatePageContent()">
+            <mat-radio-button *ngFor="let resultQualityValue of resultQualityValues"
```

Comment:
Not sure if radio buttons are the way to go here. As a rule of thumb: Somewhere between 4 and 6 is the line where a `select` is preferred as radio buttons take up much more space.